.eslintrc.js

```diff
@@ -0,0 +1,93 @@
+module.exports = {
```

Comment:
Please set up eslint the exact same way as the kronos app.  If you need to update the SCBD eslint let me know and I will give access.

Reply:
I try to same as kronos. but tomorrow i will spent more time on eslint.